### DISCUSSION TITLE
Write Access for PR - please?

### DISCUSSION THREAD

**Discussion by JoeyOverby** [bmaltais/kohya_ss#2815] [General]
Hello @bmaltais , 
I found some workarounds for my M3 Max (Macos) and was trying to create a PR to give people the option to download the files, but I don't have write access to do this. Would you please add me as a contributor so that I can at least create a PR to show that I found a solution? (And while it's not a great solution, I figured that it may help you or others on how to write it better and help with some of the Mac issues people have been having). 

I'm happy to provide references if needed (I'm a Software Engineer for almost 20 years) and using my real name so I won't be trying to mess anything up. 

Sincerely, 
Joey Overby

**Comment by bmaltais** [comment]
Usually this is what you need to do:

1. Create a fork of the repo
2. clone the forked repo locally
3. create a branch
4. modify the content of the files
5. Commit your changes to your github fork
6. create a Pull Request on this repo using your fork/branch as the source and this repo/dev (or sd3-flux.1) branch as the target

This will allow you to contribute changes to this repo.

**Reply by JoeyOverby** [reply]
Got it - thank you.
